content/en/docs/instrumentation/java/automatic.md

```diff
@@ -9,7 +12,7 @@ to any Java 8+ application. It dynamically injects bytecode to capture telemetry
 from many popular libraries and frameworks. It can be used to capture telemetry
 data at the "edges" of an app or service, such as inbound requests, outbound
 HTTP calls, database calls, and so on. To instrument application code in your
-app or service, use [Manual Instrumentation](../manual_instrumentation)
```

Comment:
Should these be `relref` links? Given that the build didn't break in `main` when these were integrated I wonder if that's why

Reply:
They don't have to be `relref`s.

Relref provides hugo "compile-time" checking (useful when running serve or build), as well as an extra CI link checking step. If the build is green, that's because all (internal) links are valid.